helm/kube-prometheus/values.yaml

```diff
@@ -431,3 +431,7 @@ prometheus:
 # Select Deployed DNS Solution
 deployCoreDNS: false
 deployKubeDNS: true
+
+##Custom Labels to be added to Prometheus Rules ConfigMap
+##
+additionalRulesConfigMapLabels: {}
```

Comment:
please add an example  explaining how to use this label

Reply:
I added an example. I'm not positive this is what you're looking for, since none of the other Helm charts have an example for this value.